public/src/components/costToShipt.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
I think you have a typo in the filename: costToShipt
It would be better to name all your test files to map the component file names exactly, like DeliverTo.jsx -> DeliverTo.test.jsx to be consistent (instead of deliverto.test.jsx, not a big deal tho)